src/OrchardCore.Modules/OrchardCore.Facebook/ScriptsMiddleware.cs

```diff
@@ -43,23 +44,19 @@ public async Task Invoke(HttpContext httpContext)
                     if (!string.IsNullOrWhiteSpace(settings?.AppId))
                     {
                         var options = $"{{ appId:'{settings.AppId}',version:'{settings.Version}'";
-                        if (string.IsNullOrWhiteSpace(settings.FBInitParams))
-                        {
-                            options = string.Concat(options, "}");
-                        }
-                        else
-                        {
-                            options = string.Concat(options, ",", settings.FBInitParams, "}");
-                        }
+                        options = string.IsNullOrWhiteSpace(settings.FBInitParams)
+                            ? string.Concat(options, "}")
+                            : string.Concat(options, ",", settings.FBInitParams, "}");
+
                         script = $"window.fbAsyncInit = function(){{ FB.init({options});}};";
                     }
                 }
 
                 if (script != null)
                 {
                     var bytes = Encoding.UTF8.GetBytes(script);
-                    var cancellationToken = httpContext?.RequestAborted ?? CancellationToken.None;
-                    await httpContext.Response.Body.WriteAsync(Encoding.UTF8.GetBytes(script).AsMemory(0, bytes.Length), cancellationToken);
+                    await httpContext.Response.Body.WriteAsync(Encoding.UTF8.GetBytes(script).AsMemory(0, bytes.Length), httpContext.RequestAborted);
```

Comment:
The original code checked both `httpContext` and `httpContext.RequestAborted)` for `null`. The latter is now missing, so I don't think the rest of the changes are needed either (apart from the ternary).

Reply:
`RequestAborted` is not nullable, the `??` operator was there because `HttpContext` was nullable before this change